haproxy.cfg

```
@@ -21,9 +21,12 @@ frontend router
   acl is_activitypub_request req.hdr(content-type) -i "application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\""
   use_backend streaming if { path_beg /api/v1/streaming }
   use_backend mastodon if is_activitypub_request
-  use_backend elk if { path_beg /elk }
-  use_backend rewrite-to-elk-svrname if { path '/explore' } || { path '/list' } || { path '/local' } || { path '/notifications' } || { path '/public/local' } || { path_beg /@ } || { path_beg /tags }
-  use_backend rewrite-to-elk-plain if { path / } || { path /bookmarks } || { path /conversations } || { path /favourites } || { path /home } || { path /public } || { path /publish } || { path '/search' } || { path_beg /settings }
+  use_backend elk if { path_beg /_nuxt }
+  use_backend elk if { path_beg /api/${ROUTER_HOSTNAME-stage.moztodon.nonprod.webservices.mozgcp.net} }
+  use_backend mastodon if { path_beg /api }
+  use_backend rewrite-publish if { path '/publish' } || { path_beg /settings/ }
+  use_backend elk if { path / } || { path '/compose' } || { path /settings } || { path /bookmarks } || { path '/conversations' } || { path '/favourites' } || { path '/home' } || { path '/public' } || { path '/search' }
+  use_backend elk if { path_beg /settings }  || { path '/explore' } || { path '/list' } || { path '/local' } || { path '/notifications' } || { path '/public/local' } || { path_beg /@ } || { path_beg /tags }
```

```suggestion
  use_backend rewrite-to-elk-svrname if { path '/explore' } || { path '/list' } || { path '/local' } || { path '/notifications' } || { path '/public/local' } || { path_beg /@ } || { path_beg /tags }
```
Guessing `elk` as the backend was a typo?

---

HAProxy didnt like a line longer then 240 characters 🤣  this was on purpose.

---

ah, so the `backend rewrite-to-elk-svrname` section can also go away?

---

Ah yep. let me burn that.

---

Done.